stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh

```diff
@@ -30,6 +30,7 @@ Additional environment variables:
     DOCKER_IMG: Docker image to use for building (Default: stratumproject/build:build)
     RELEASE_BUILD: Optimized build with stripped symbols (Default: false)
     BAZEL_CACHE: Path to Bazel cache (Default: <empty>)
+    BSP: Path to optional BSP package directory (Default: <empty>)
```

Comment:
This is an env var because this script does no support/use flag style arguments for the other parameters.